pkg/oc/admin/diagnostics/diagnostics/cluster/network/in_pod/util/log.go

```diff
@@ -206,3 +206,40 @@ func (l *LogInterface) getConfigFileForService(serviceName, serviceArgs string)
 	}
 	return string(out[:])
 }
+
+// ContainerID is a type that identifies a container.
+type ContainerID struct {
```

Comment:
Any particular reason we're copy&pasting this from upstream? The functionality is *exactly* the same.

Reply:
> quasi public packages

That's sufficiently strong option.